libkbfs/folder_branch_ops.go

```diff
@@ -6195,7 +6195,7 @@ func (fbo *folderBranchOps) GetUpdateHistory(ctx context.Context,
 		}
 		updateSummary := UpdateSummary{
 			Revision:  rmd.Revision(),
-			Date:      time.Unix(0, rmd.data.Dir.Mtime),
+			Date:      rmd.localTimestamp,
```

Comment:
We are using `localTimestamp` here because the server timestamp is not trusted? Although this history can include updates from other devices, so there can be clock issues, and the time when the updates are merged to master also may matter. Do you think we should include server timestamp in addition to the local one?